doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -1819,23 +1845,83 @@ It is expected that the restore command is run by the xroad user.
+If it is absolutely necessary to restore the system from a backup made on a different security server, the forced mode
+of the restore command can be used with the –F option together with unencrypted backup archive flags. For example (all on one line):
 
     /usr/share/xroad/scripts/restore_xroad_proxy_configuration.sh \
-    -F –f /var/lib/xroad/backup/conf_backup_20140703-110438.tar
+    -F -P –f /var/lib/xroad/backup/conf_backup_20140703-110438.tar
+
+In case backup archives were encrypted they have to be first unencrypted in external safe environment and then securely
+transported to security server filesystem.
 
 ### 13.3 Automatic Backups
 
-By default the Security Server backs up its configuration automatically once every day. Backups older than 30 days are automatically removed from the server. If needed, the automatic backup policies can be adjusted by editing the `/etc/cron.d/xroad-proxy` file.
+By default the Security Server backs up its configuration automatically once every day. Backups older than 30 days are
+automatically removed from the server. If needed, the automatic backup policies can be adjusted by editing the
+`/etc/cron.d/xroad-proxy` file.
+
+### 13.4 Backup Encryption Configuration
+
+Back up encryption is initially turned off. To turn encryption on configuration must be overridden in the file `/etc/xroad/conf.d/local.ini`, in the `[proxy]` section.
+
+    [proxy]
+    backup-encrypted=false
+    backup-public-key-path=/etc/xroad/backupkeys
+
+Backups are always signed. To turn on backup encryption add or change the `backup-encrypted` key to true. If needed also 
+change the folder where additional backup encryption keys are stored. Default is `/etc/xroad/backupkeys`.
+
+Backup encryption keys folder must contain only GPG public key files. In addition to security servers public key backup
+archives are encrypted with all the keys in this folder. It is recommended to use at least one additional public key,
+otherwise the backups will be unusable in case security servers private key is lost. It is up to security servers administrator
+to check that private keys used are sufficiently strong, there are no automatic checks.
+
+Additional keys for backup encryption should be generated and stored outside security server in a secure environment.
+After gpg keypair has been generated, public key can be exported to a file (backupadmin@organisation.eu is the name of the
+key being exported) using this command:
+
+    gpg --output backupadmin.publickey --armor --export backupadmin@organisation.eu
+
+Resulting file `backupadmin.publickey` should be moved to security server and copied to backup encryption keys folder.
+Transfer of the public key does not have to be secure but administrator has to make sure that file is not changed during
```

Comment:
It would be good to specify the file permissions that must be set to the key on the Security Server and provide a command how to set them.